pages/index.vue

```diff
@@ -41,62 +28,8 @@ import { GeneralLink } from '~/constants/appLinks'
 export default class LandingPage extends QiskitPage {
   routeName = 'qiskit-landing-page'
 
-  mosaicSectionTitle = 'What can Qiskit do'
-  mosaicSectionDescription = `Qiskit accelerates the development of quantum applications by
+  qiskitCapabilitiesSectionTitle = 'What Can Qiskit Do'
+  qiskitCapabilitiesSectionDescription = `Qiskit accelerates the development of quantum applications by
```

Comment:
We might not need these variables if we decide to drop the props from the **TheQiskitCapabilitiesSection** component:

```suggestion
```